packages/editor/src/plugins/text/components/text-editor.tsx

```diff
@@ -163,11 +166,15 @@ export function TextEditor(props: TextEditorProps) {
 
       {focused ? <LinkControls /> : null}
 
-      {showSuggestions ? (
-        <SlateOverlay width={620}>
-          <Suggestions {...suggestionsProps} />
-        </SlateOverlay>
-      ) : null}
+      <ModalWithCloseButton
+        className="top-8 max-h-[90vh] w-auto min-w-[700px] translate-y-0 overflow-y-scroll pt-0"
+        isOpen={showSuggestions}
+        setIsOpen={setShowSuggestions}
+        altText={editorStrings.addPluginsModal.altText}
+      >
+        {/* TODO: Remove added text plugin if user closes modal without selecting text plugin */}
```

Comment:
maybe now it's easier to not create a text plugin in the first place?

Reply:
If that's possible, I'd welcome it :)

Reply:
The whole suggestions logic still lives in the text plugin, investigating how to avoid that but I do believe it's out of scope for this ticket

Reply:
> The whole suggestions logic still lives in the text plugin

After the change of the issue there should be IMHO a refactoring to move the code out into the rows plugin. still integrating a text plugin might be a reason for bugs in the future...